### DISCUSSION TITLE
Problems setting up LDAP filter

### DISCUSSION THREAD

**Discussion by schmunk42** [requarks/wiki#5835] [Help / Questions] [answered]
I am trying to configure wikijs with LDAP auth.
I can connect to the LDAP server (we have several apps running with LDAP login).

But I get **You are not authorized to login.** when using my LDAP Account.

I tried to setup the Search filter, but without luck

Wikijs default:  `(uid={{username}})`

Config from Gitlab:
```
LDAP_USER_FILTER: '(objectClass=inetOrgPerson)'
LDAP_UID: 'uid'
```

Config from Symfony App:
```
usernameAttribute: uid
filter: (&(objectClass=inetOrgPerson))
```

So it looks like I need to combine the two, but weren't able to figure out how!?

**Comment by NGPixel** [comment, marked as answer]
Do you have **self-registration** enabled?

**Reply by schmunk42** [reply]
> Do you have **self-registration** enabled?

No.

Addon to inital posting:
I tried to combine the two criteria like so `(&(objectClass=inetOrgPerson)(uid={{username}}))`, but without luck.

**Reply by NGPixel** [reply]
You need to enable self-registration on the ldap strategy.

**Reply by schmunk42** [reply]
I'll try that tomorrow. 

But why? I acutally don't want self-registration, especially with LDAP.

**Reply by NGPixel** [reply]
self-registration simply means that an authorized user from LDAP can be created once it reaches wiki.js, it doesn't allow someone to create a LDAP account...

**Reply by schmunk42** [reply]
Thank you very much!
Side question: What would be the use-case to have this option disabled?
